### PR TITLE
Elo ranking for fifa

### PR DESCRIPTION
**Pull request by stevenjob**

### CODE REVIEW

File: PS4History.py [hunk 1]
```diff
@@ -166,6 +167,39 @@ def allow_user(u):
 
         return stats # { mode: { user: { [stat]: int ... }, ... } }
 
+
+
+    def summary_elo(self, channel, name = None, year = None):
+
+        ## TODO handle non fifa stuff here
+        ## or hardcode to fifa2 channel
```
Comment (stevenjob) [inline PS4History.py:175]: need @bobrippling  to fix this part to work for all game types plz

File: PS4Elo.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+        individual_ranking_delta = getRankingDeltaForGame(game, players)
+
+        for team in game.teams:
+            for player in map(lambda playerId: getPlayerFromId(players, playerId), team):
+                player.games_played += 1
+
+                scrub_modifier = calculateScrubModifier(player, game)
+
+                ## if the player wins and is sotm they get bonus points TODO fix
```
Comment (stevenjob) [inline PS4Elo.py:149]: lol

File: PS4History.py (hunk 1 above)
Comment (bobrippling) [reply PS4History.py:175]: Sortid :+1:

File: PS4Elo.py (hunk 2 above)
Comment (bobrippling) [reply PS4Elo.py:149]: lol indeed, i've fixed it to only apply if a player was on the losing team but if you want it the other way, feel free to fly in a PR

**Comment by bobrippling** [comment]
Code's solid, my changes were mainly matching the existing style and integrating with ps4bot more

**Pull request merged**